.changes/v3.8.0/899-deprecations.md

```diff
@@ -0,0 +1,2 @@
+* Deprecated `resource/vcd_catalog_item` in favor of `resource/vcd_catalog_vapp_template` [GH-899]
```

Comment:
Let's make sure that we have a fully functioning VM creation with the new vApp template before the release, or else this deprecation will have to be postponed

Reply:
Good point, will remove the deprecations.